Detectors/CPV/simulation/src/Digitizer.cxx

```diff
@@ -27,53 +28,48 @@ using namespace o2::cpv;
 //_______________________________________________________________________
 void Digitizer::init()
 {
-  if (!mCalibParams) {
-    if (o2::cpv::CPVSimParams::Instance().mCCDBPath.compare("localtest") == 0) {
-      mCalibParams.reset(new CalibParams(1)); // test default calibration
-      LOG(INFO) << "[CPVDigitizer] No reading calibration from ccdb requested, set default";
-    } else {
-      LOG(INFO) << "[CPVDigitizer] can not get calibration object from ccdb yet. Using default";
-      mCalibParams.reset(new CalibParams(1)); // test default calibration
-      //      o2::ccdb::CcdbApi ccdb;
-      //      std::map<std::string, std::string> metadata; // do we want to store any meta data?
-      //      ccdb.init("http://ccdb-test.cern.ch:8080");  // or http://localhost:8080 for a local installation
-      //      mCalibParams = ccdb.retrieveFromTFileAny<o2::cpv::CalibParams>("CPV/Calib", metadata, mEventTime);
-      //      if (!mCalibParams) {
-      //        LOG(FATAL) << "[CPVDigitizer] can not get calibration object from ccdb";
-      //      }
+  LOG(INFO) << "CPVDigitizer::init() : CCDB Url = " << o2::cpv::CPVSimParams::Instance().mCCDBPath.data();
```

Comment:
I am wondering why each detector should have this path configurable (separately)? In principle there is already a global configurable `DigiParams` that can take this path.
I understand that several detectors are doing it like here, but eventually we should converge on a single source. (Not asking this necessarily now. I'll open a ticket to organize a campaign for all detectors.)

Reply:
Dear Sandro, 

I am also wondering same question. Next PR I'll change it to DigiParams. Also if I do `cat o2sim_configuration.ini | grep -i ccdb` to some simulation config, I see 4 times ccdb path configured 4 times:
```
ccdb=http://ccdb-test.cern.ch:8080
mCCDB=http://ccdb-test.cern.ch:8080
mCCDBPath=localtest
mCCDBPath=ccdb-test.cern.ch:8080
```
 Just to inform you.